app/javascript/src/MarkdownViewer/ror_components/MarkdownViewer.jsx

```
@@ -0,0 +1,25 @@
+ *
+ * Processing happens in the consuming components:
+ * - Server component: processes markdown server-side (heavy libs stay on server)
+ * - Client component: processes markdown client-side (heavy libs go to client)
+ *
+ * This pattern ensures the viewer itself has minimal bundle impact.
+ */
+const MarkdownViewer = ({ processedHtml, className }) => {
+  return (
+    <div
+      className={`${style.markdownContent} ${className || ''}`}
+      dangerouslySetInnerHTML={{ __html: processedHtml }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)